app_dart/lib/src/model/proto/internal/scheduler.proto

```diff
@@ -57,7 +57,17 @@ message Target {
     // Name of the LUCI recipe to use for the builder.
     optional string recipe = 13;
     // Additional tag for the target.
-    repeated string tags = 14;
```

Comment:
Can we link to the LUCI docs on what a tag is?

Reply:
I am hesitate to do that now seems these tags are not feed into the luci yet. Should we wait for that once we actually feed this tags into luci?